### DISCUSSION TITLE
syncing

### DISCUSSION THREAD

**Discussion by Tresslin** [Chia-Network/chia-blockchain#16675] [Support]
Hi im using Chia now for 2 days but its still not syncing I plotted 7 already but the syncing is not starting

**Comment by BrandtH22** [comment]
Hey @Tresslin , as a heads up we can generally provide more timely and thorough support on our discord server (https://discord.gg/chia).

As for node syncing I will generally check these in this order:

1. Am I connected to peers? 
- Using the gui you can click on the full node icon in the top right to see your peer connections and the data you are downloading from them. If you are not connecting to peers I recommend going through these steps: https://docs.chia.net/node-syncing

2. Is my blockchain db on a fast ssd?
- If you are using a slower storage medium for the database (like an external HDD) then you might want to move the database to a faster drive like an SSD following these steps: https://docs.chia.net/faq/#how-do-i-move-the-chia-blockchain-database-to-another-drive

3. If the above fails is not working or applicable then do I have any errors in my logs?
- Here is a quick guide to find the logs and extract data from them: https://docs.chia.net/check-if-things-are-working/

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
